framework/src/loops/ComputeUserObjectsThread.C

```diff
@@ -191,6 +198,51 @@ ComputeUserObjectsThread::onInternalSide(const Elem * elem, unsigned int side)
   }
 }
 
+void
+ComputeUserObjectsThread::onInterface(const Elem * elem, unsigned int side, BoundaryID bnd_id)
+{
+
+  // Pointer to the neighbor we are currently working on.
+  const Elem * neighbor = elem->neighbor_ptr(side);
+
+  // Get the global id of the element and the neighbor
+  const dof_id_type elem_id = elem->id(), neighbor_id = neighbor->id();
+
+  if (!_interface_user_objects.hasActiveBoundaryObjects(bnd_id, _tid))
+  {
+    return;
+  }
+
+  if (!((neighbor->active() && (neighbor->level() == elem->level()) && (elem_id < neighbor_id)) ||
+        (neighbor->level() < elem->level())))
+  {
+    return;
+  }
+
+  _fe_problem.prepareFace(elem, _tid);
+  // TODO next line does not produce nay effect
```

Comment:
Is this your comment? If it doesn't do anything, remove the commented lines. It may not have any effect because `reinitElemFace` has already been called...looks like this happens in `onBoundary`